conans/model/layout.py

```diff
@@ -129,6 +149,11 @@ def imports_folder(self):
 
         return os.path.join(self._base_imports, self.imports)
 
+    @property
+    def imports_path(self) -> Path:
```

Comment:
This one disappears in 2.0, better remove it.

Reply:
removed